apps/safe-claiming-app/src/widgets/__tests__/index.test.ts

```diff
@@ -0,0 +1,32 @@
+import { _getProposalNumber, _getProposalTitle } from "../Snapshot"
+
+describe("SnapshotWidget", () => {
```

Comment:
Nice that there is a test.

If we keep this logic and the variants keep growing this will help alot 👍

Reply:
hats off to @iamacook  . I only ported this part.